ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/tree/member/services/TreeMemberRemoverImpl.java

```diff
@@ -16,7 +16,7 @@ public TreeMemberRemoverImpl(MemberRepository memberRepository) {
 		this.memberRepository = memberRepository;
 	}
 
-	public void tryRemovingMember(String memberId) {
```

Comment:
I am not sure I would call this 'try...' since that always applies to everything. If you need to be sure it was successful, you ask feedback.